onnxscript/rewriter/llama_rule_sets.py

```diff
@@ -2,13 +2,73 @@
 # Licensed under the MIT License.
 from __future__ import annotations
 
+import numpy as np
+import onnx.numpy_helper
+
 import onnxscript.ir as ir
 import onnxscript.rewriter.no_op as no_op
 import onnxscript.rewriter.pattern as orp
 
 op = orp.onnxop
 
 
+class CastIdentity(orp.RewriteRuleAsClass):
```

Comment:
I am surprised if ORT doesn't implement this already. Still, I can see it being useful here.

Reply:
It probably does. But my patterns work like layers. You need this one so that the next one cast triggers.